dashboard/db/migrate/20241024171901_created_submitted_projects.rb

```diff
@@ -0,0 +1,13 @@
+class CreatedSubmittedProjects < ActiveRecord::Migration[6.1]
+  def change
+    # use utf8mb4 to support emojis
+    create_table :submitted_projects, charset: 'utf8mb4', collation: 'utf8mb4_bin' do |t|
+      t.integer :project_id
+      t.string :description
+      t.datetime :declined_at
```

Comment:
Might want to index on this as well since it'll be part of the lookups you'll be doing?

Reply:
Good idea. Will update.